src/main/groovy/com/github/jk1/license/render/LicenseDataCollector.groovy

```diff
@@ -0,0 +1,79 @@
+package com.github.jk1.license.render
+
+import com.github.jk1.license.License
+import com.github.jk1.license.LicenseReportPlugin
+import com.github.jk1.license.ModuleData
+
+class LicenseDataCollector {
+
+    protected static List<String> singleModuleLicenseInfo(LicenseReportPlugin.LicenseReportExtension config, ModuleData data) {
+        def info = multiModuleLicenseInfo(config, data, false)
+        def moduleUrl = info.moduleUrls?.last()
+        def license = info.licenses?.last()
+        def moduleLicense = license?.name
+        def moduleLicenseUrl = license?.url
+
+        [moduleUrl, moduleLicense, moduleLicenseUrl]
+    }
+
+    protected static MultiLicenseInfo multiModuleLicenseInfo(
+        LicenseReportPlugin.LicenseReportExtension config, ModuleData data, boolean alwaysCheckLicenseFiles = true) {
+        MultiLicenseInfo info = new MultiLicenseInfo()
+
+        data.manifests.each {
+            if (it.url) {
+                info.moduleUrls << it.url
+            }
+            if (it.license) {
+                if (it.license.startsWith('http')) {
+                    info.licenses << new License(url: it.license)
+                } else {
+                    info.licenses << new License(name: it.license)
+                }
+            }
+        }
+        data.poms.each {
+            if (it.projectUrl) {
+                info.moduleUrls << it.projectUrl
+            }
+            if (it.licenses) {
+                info.licenses.addAll(it.licenses)
+            }
+        }
+
+        // check just here for backward compatibility reason of "simple" json-renderer
+        // think about removing this at all and risk, that the simple-renderers gets a different result in case
+        // several licenses are available
+        if (info.licenses.isEmpty() || !info.licenses.last().url || alwaysCheckLicenseFiles) {
+            data.licenseFiles.each {
+                it.files.each {
+                    String moduleLicense = null
+                    String moduleLicenseUrl = null
+                    def text = new File(config.outputDir, it).text
+                    if (text.contains('Apache License, Version 2.0')) {
```

Comment:
Why hardcode these (and only these) licenses? I think a better fix would be to turn on `LicenseNormalizerBundle` by default.

Reply:
Think so too. I'll add that in the next PR (tomorrow). In this one, I focus on having a multi-json-renderer and try not to break backward compatibility.